tests/unit_tests/ec2/test_cloud.py

```diff
@@ -70,9 +70,16 @@ class TestEC2:
                 "noble",
                 ImageType.PRO_FIPS,
                 False,
-                "ubuntu-pro-fips*/images/hvm-ssd-gp3/ubuntu-noble-24.04-*",
+                "ubuntu-pro-fips-server*/images/hvm-ssd-gp3/ubuntu-noble-24.04-*",
                 id="pro-fips-lts",
             ),
+            pytest.param(
+                "jammy",
+                ImageType.PRO_FIPS_UPDATES,
+                False,
+                "ubuntu-pro-fips-updates-server*/images/hvm-ssd/ubuntu-jammy-22.04-*",
```

Comment:
likewise, you can drop the `*` from `ubuntu-pro-fips-updates-server*`

Reply:
done